EventsExpress.Core/Services/EventScheduleService.cs

```diff
@@ -86,6 +91,25 @@ public async Task<Guid> Edit(EventScheduleDto eventScheduleDTO)
             return eventScheduleDTO.Id;
         }
 
+        public async Task<Guid> Delete(Guid id)
+        {
+            var eventSchedule = Context.EventSchedules.Find(id);
+            if (eventSchedule == null)
+            {
+                throw new EventsExpressException("Not found");
+            }
+
+            var result = Delete(eventSchedule);
+            if (result.Id != id)
```

Comment:
This `if` makes no sense here. Please remove it.

Reply:
Ok. I've already noticed and removed this when I was writing unit tests for this method. These changes will be delivered in the next commit when I finish writing all unit tests